pkg/apis/runoncedurationoverride/v1/override_types.go

```diff
@@ -50,16 +50,36 @@ type RunOnceDurationOverrideCondition struct {
 // +genclient
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-
+// +k8s:openapi-gen=true
+// +kubebuilder:storageversion
```

Comment:
If I understand correctly, CRD is auto generated by kubebuilder by using these tags?

Reply:
Yeap. These tags help to shape the CRD.